libraries/radpdfprocessing/features/embedded-file-streams/embedded-file-streams.md

```diff
@@ -30,18 +32,43 @@ RadFixedDocument stores the integrated files in an **EmbeddedFilesCollection** a
 
 {{endregion}}
 
->note RadPdfProcessing  provides support for embedding of [ZUGFeRD](https://de.wikipedia.org/wiki/ZUGFeRD) (acronym for Zentraler User Guide des Forums elektronische Rechnung Deutschland) invoices.
+>caution **DuplicatedEmbeddedFileNameException** is thrown when adding an embedded file with a name that is already added to the collection.
+
+#### Attachments section in Adobe 
+![Embedded Files in a PDF document](images/embedded_files_0.png)
+
+### Creating an Embedded Electronic (ZUGFeRD) Invoice
+
+RadPdfProcessing provides support for embedding of [ZUGFeRD](https://de.wikipedia.org/wiki/ZUGFeRD) (acronym for Zentraler User Guide des Forums elektronische Rechnung Deutschland) invoices.
 
-#### **[C#] Add ZUGFeRD invoide**
+#### **[C#] Add ZUGFeRD invoice**
 
 {{region cs-radpdfprocessing-embedded-file-add-zugferd-invoice}}
 
-byte[] bytes = File.ReadAllBytes(@"..\zugferd-invoice.xml");
-document.EmbeddedFiles.AddZugferdInvoice(bytes);
+            RadFixedDocument document = new RadFixedDocument();
+            using (RadFixedDocumentEditor editor = new RadFixedDocumentEditor(document))
+            {
+                editor.CharacterProperties.TrySetFont(new System.Windows.Media.FontFamily("Calibri"));
+                editor.InsertRun("Hello PDF/A");
```

Comment:
Instead of "Hello PDF/A" we can insert "PDF/A-3B Complaint Invoice" or something like this.